docs/source/managed-federation/overview.mdx

```diff
@@ -3,7 +3,7 @@ title: Managed federation overview
 sidebar_title: Overview
 ---
 
-[Apollo Studio](https://www.apollographql.com/docs/studio/) provides free **managed federation** features for data graphs that use Apollo Federation.
+[Apollo Studio](https://www.apollographql.com/docs/studio/federated-graphs/) provides free **managed federation** features for data graphs that use Apollo Federation.
```

Comment:
Do you think it might make sense to leave the top level link on `Apollo Studio` and add this link on the words `managed federation`?

also cc @StephenBarlow who probably has lots of insight into how our docs interconnect

Reply:
@glasser that seems kind of confusing to me, honestly. In this context, if "Apollo Studio" isn't going to take me to the specific article about working with federated graphs in Studio, the only other place I would expect it to take me is to Studio itself. Like if I'm reading about Federated graphs, I would expect a reference to the Studio docs to take me directly to the part that is relevant to working with Federated graphs.

And also I can see how having both links here on this line might be useful. So I guess I could go either way on this one. Not strongly opinionated. :)